src-11ty/_includes/base.html

```diff
@@ -53,10 +53,8 @@
       }
 
       @media all and (max-width: 500px) {
-        .layout {
-          justify-content: flex-start;
```

Comment:
This keeps the card now centered on mobile. Is that what we want?

Reply:
![Screen Shot 2021-02-17 at 08 22 07](https://user-images.githubusercontent.com/1444314/108234036-3dbe9580-70f9-11eb-888a-6bd4b5b42cad.png)

Reply:
I liked it most for the redirect at least, but your screen is way higher than mine 😄

Reply:
This is just on my mac ofc, not a real phone. Ok let's keep it .